client/components/domains/domain-suggestion-flag/style.scss

```diff
@@ -0,0 +1,7 @@
+.domain-suggestion__content {
+	> h3 {
+		.flag {
+			margin-left: 5px;
```

Comment:
In my very humble opinion, adding `vertical-align: middle;` here makes the label line up with the domain "properly" (otherwise, the label is slightly below the domain).